src/components/ListedSpecies/BandingColorSelect.jsx

```diff
@@ -0,0 +1,77 @@
+import React, { useEffect, useState } from 'react';
+import { IconContext } from 'react-icons';
+import { BsFillCircleFill } from 'react-icons/bs';
+import { useFormContext } from 'react-hook-form';
+import { chakraComponents, Select } from 'chakra-react-select';
+import { chakra } from '@chakra-ui/react';
+import PropTypes from 'prop-types';
+import BandColors from '../../common/BandColors';
+
+// setup custom rendering to show the band color in color select dropdown
+const customComponents = {
+  // eslint-disable-next-line react/prop-types
+  Option: ({ children, ...props }) => (
+    // eslint-disable-next-line react/jsx-props-no-spreading
+    <chakraComponents.Option {...props}>
+      {/* eslint-disable-next-line react/prop-types */}
+      <chakra.span
+        marginRight={2}
+        borderWidth="1px"
+        // eslint-disable-next-line react/prop-types
+        borderColor={props.data.value === 'W' ? 'gray.600' : props.data.realColor}
+        borderRadius="3xl"
+      >
+        {/* eslint-disable-next-line react/prop-types */}
```

Comment:
Nit: group `eslint-disable` into a single one wrapping the entire `customComponents` object:

```
/* eslint-disable react/prop-types, react/jsx-props-no-spreading */
const customComponents = { ... }
/* eslint-enable react/prop-types, react/jsx-props-no-spreading */
```